### DISCUSSION TITLE
Treesitter and LSP, should I install those myself?

### DISCUSSION THREAD

**Discussion by kresdjan** [nvim-mini/mini.nvim#274] [Q&A] [answered]
Forgive my ignorance. But I am supposed to install  treesitter [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) and LSP [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) myself, or is something a functionality similar to Mason or Lazy that come with one of the modules. that will install the language grammars and LSPs?

**Comment by echasnovski** [comment, marked as answer]
Don't worry, we all have been learning some day.

Yes, **if** users need tree-sitter or LSP powered functionality they are expected to install it themselves. This is not (yet) a pre-configured distribution like LazyVim.
